src/Components/Pages/Form/Form.tsx

```diff
@@ -49,9 +50,28 @@ export const Form: React.FC = () => {
 
   const NUM_STEPS = !bldgData ? 5 : bldgData?.unitsres > 10 ? 5 : 6;
 
+  const formStepRefs = [
+    useRef<HTMLFieldSetElement | null>(null),
+    useRef<HTMLFieldSetElement | null>(null),
+    useRef<HTMLFieldSetElement | null>(null),
+    useRef<HTMLFieldSetElement | null>(null),
+    useRef<HTMLFieldSetElement | null>(null),
+    useRef<HTMLFieldSetElement | null>(null),
+  ];
+
   const navigate = useNavigate();
 
   const handleSubmit = () => {
+    const firstUnansweredIndex = Object.values(localFields).findIndex(
+      (x) => x === null
+    );
+    if (firstUnansweredIndex >= 0 && firstUnansweredIndex <= NUM_STEPS - 1) {
+      setShowErrors(true);
+      formStepRefs[firstUnansweredIndex].current?.scrollIntoView({
```

Comment:
this looks so neat. the site header does cut off a little bit of the form step it jumps to, but I don't think it's a big deal unless it is brought up again in VQA
https://www.loom.com/share/3cb7dfdc0ab94e189e7ad735993203eb

Reply:
Thanks for catching this. I think the fixed header on page might be throwing things off a bit. Just added `scroll-margin-top` to the form steps to offset that and now it's positioned with the full box in view